spec/jobs/appointment_notification/worker_spec.rb

```diff
@@ -3,11 +3,16 @@
 RSpec.describe AppointmentNotification::Worker, type: :job do
   describe "#perform" do
     before do
-      Flipper.enable(:appointment_reminders)
+      Flipper.enable(:notifications)
       allow(Statsd.instance).to receive(:increment).with(anything)
     end
 
-    let(:reminder) { create(:appointment_reminder, status: "scheduled", message: "sms.appointment_reminders.missed_visit_whatsapp_reminder") }
+    let(:notification) {
+      create(:notification,
+        subject: create(:appointment),
+        status: "scheduled",
+        message: "sms.notifications.missed_visit_whatsapp_reminder")
```

Comment:
This should also probably be changed back to appointment_reminders.

Reply:
oh ya fixed